snapcraft/internal/errors.py

```diff
@@ -531,3 +532,20 @@ class InvalidExtractorValueError(MetadataExtractionError):
 
     def __init__(self, path: str, extractor_name: str) -> None:
         super().__init__(path=path, extractor_name=extractor_name)
+
+
+class SnapcraftPluginCommandError(SnapcraftError):
+    """Command executed by a plugin fails."""
+
+    fmt = (
+        'The command {command!r} run for the {part_name!r} part has failed.\n'
+        'Exited with exit code {exit_code}.\n'
+        'Verify that the part is using the correct parameters and try again.'
```

Comment:
Channeling my inner Leo, this does not meet the agreed-upon error message format which is:

```
<what>: <why>\n
<how to fix>
```

It seems to be

```
<what>\n
<why>\n
<how to fix>
```